### PR TITLE
Pass start.sh first argument as a command for bash

### PR DESCRIPTION
**Pull request by 6uliver**
Signed-off-by: Roland Németh <6uliver@gmail.com>

### CODE REVIEW

**Comment by nathanleclaire** [comment]
@6uliver Thanks for the PR.

Can you explain a bit more the motivations please?

**Comment by 6uliver** [comment]
I'm using a console emulator (http://cmder.net/) on Windows, and I wanted to set Toolbox start script for startup task. It's convenient for me to fully setup dev environment, for example setting a local domain in drivers/etc/host file for Docker's VM. I managed to do that with "docker-machine ip", but I have to run it somehow, so it would be practical to pass a command to start.sh.

**Comment by nathanleclaire** [comment]
OK.  Why not pass `$@` (all remaining arguments) instead of `$1`?

@tianon Can you please take a look at this and tell me what you think?  Thanks man.

File: windows/start.sh [hunk 1]
```diff
@@ -68,4 +68,10 @@ docker () {
 }
 export -f docker
 
-exec "${BASH}" --login -i
+if [ $# -eq 0 ]; then
+  echo "Start interactive shell"
+  exec "$BASH" --login -i
+else
+  echo "Start shell with command"
+  exec "$BASH" -c "$1"
```
Comment (tianon) [inline windows/start.sh]: Hmm, `$*` might actually be more appropriate than either `$1` or `$@` here, so that we're explicit about our space expansion.

Comment (tianon) [inline windows/start.sh]: But other than that, this seems generally sane to me!

**Comment by 6uliver** [comment]
You're right, I fixed it.

**Comment by tianon** [comment]
:+1:

**Comment by nathanleclaire** [comment]
LGTM

**Comment by 6uliver** [comment]
Thank you!

**Comment by nathanleclaire** [comment]
image

**Pull request merged**